apypie/api.py

```diff
@@ -40,6 +48,7 @@ class Api(object):
     :param password: password to access the API
     :param client_cert: client cert to access the API
     :param client_key: client key to access the API
+    :param kerberos: use Kerberos/GSSAPI for authentication with the API
```

Comment:
Should it mention the additional dependency? Or that it ignores the `username` and `password` parameters?

Reply:
mentioned the dep